src/types/pivot_runtime.ts

```diff
@@ -15,8 +15,11 @@ export interface Pivot<T = PivotRuntimeDefinition> {
   getTableStructure(): SpreadsheetPivotTable;
   getFields(): PivotFields | undefined;
 
-  getPivotHeaderValueAndFormat(domain: StringDomainArgs): FPayload;
-  getPivotCellValueAndFormat(measure: string, domain: StringDomainArgs): FPayload;
+  parseDomain(args: Maybe<FPayload>[]): DomainArgs;
+
+  getPivotHeaderValueAndFormat(domain: DomainArgs): FPayload;
+  getPivotCellValueAndFormat(measure: string, domain: DomainArgs): FPayload;
+  getPivotMeasureValue(measure: string, domain: DomainArgs): FPayload;
```

Comment:
I'm sad, but I have to keep the domain in this function to be able to compute used cell in the super feature "Insert pivot missing cells" in Odoo